src/pretalx/mail/signals.py

```diff
@@ -8,6 +8,18 @@
 As with all event-plugin signals, the ``sender`` keyword argument will contain the event.
 """
 
+queuedmail_pre_send = EventPluginSignal()
+"""
+This signal is sent out before a ``QueuedMail`` will been sent.
```

Comment:
It might be good to explain here that this signal will only be sent for emails that will be saved, so it will not be triggered for emails that are fire-and-forget for privacy reasons, like password resets or reminders about draft proposals.